pkg/platform/packet/packet.go

```diff
@@ -250,6 +250,15 @@ func (c *config) GetExpectedNodes() int {
 func (c *config) checkValidConfig() hcl.Diagnostics {
 	var diagnostics hcl.Diagnostics
 
+	diagnostics = append(diagnostics, c.checkNotEmptyWorkers()...)
+	diagnostics = append(diagnostics, c.checkWorkerPoolNamesUnique()...)
+
+	return diagnostics
+}
+
+func (c *config) checkNotEmptyWorkers() hcl.Diagnostics {
```

Comment:
Can we please have some short comment for each function?

Reply:
@iaguis when you're addressing the review comments, could you respond to them or ideally mark them as resolved please? That helps with consecutive reviews.

Reply:
Sure